### ISSUE TITLE
HoloViews/ hvplot not working in code-server Jupyter Notebook on Binder

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/holoviews#5182]
I'm trying to create a nice repository for me and Sophia M Yang for a blog post and tutorial. I would also like to show case that the HoloViz ecosystem works with VS Code. So I added the code-server to the Binder environment.

Unfortunately it seems not to work

## Reproducible Steps

- Open https://mybinder.org/v2/gh/marcskovmadsen/hvplot_interactive/3cd8bf19dc326a76efe20e2dd3fedb7f1ff6e6af
- Open VS Code
- View Output - Jupyter to see log message
- Create a new notebook
- Make sure the kernel is `notebook`
- 

```python
print('hello')
```

```python
import hvplot.pandas
```

Notice how it took +18s to execute this cell

```python
print('hello')
```

Notice the console how you get an error message as you print hello.

```python
import holoviews as hv

hv.extension("bokeh")
```

Notice how much time it takes before the timer starts running. And that it takes a very, very long time for hv.extension('bokeh') to load.

## Log for Output - Jupyter window

[stacktrace.txt](https://github.com/holoviz/holoviews/files/7874720/stacktrace.txt)

## Additional Context

- I don't experience the same problems in "real" VS Code on my laptop.
- I experience similar problems on my Jupyter Hub for work where code-server is also running.

**Comment by maximlt** [comment]
Can pure Bokeh plots be rendered in code-server?

**Comment by MarcSkovMadsen** [comment]
Yes. There seems to be no problem in the interactive window. But the notebook is problematic.

**Comment by MarcSkovMadsen** [comment]
For me the interactivity in Code is key. It is hard for others to keep up with and compete with that environment. So need to work there.

**Comment by MarcSkovMadsen** [comment]
This is an example of Bokeh/ Holoviews plots being render in code-server @maximlt . I'm used to having full interactive with 2-way comms in VS Code on my laptop. Here it does not work. But I can get it to render in the Interactive Interpreter. In the Notebook everything gets too slow.

![image](https://user-images.githubusercontent.com/42288570/149633676-972e0793-a8ec-4cad-9c42-8858cb97379f.png)
